src/components/NewSectionSegmentPopup.jsx

```diff
@@ -0,0 +1,112 @@
+import { useState, React } from 'react';
+import {
+  Button,
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  RadioGroup,
+  Radio,
+  Stack,
+  Input,
+  Text,
+} from '@chakra-ui/react';
+import { AddIcon } from '@chakra-ui/icons';
+import Section from './Section';
+
+function NewSectionSegmentPopup(onAddSegment) {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [value, setValue] = useState('0');
+  const [volInput, setVolInput] = useState('');
+  const [adminName, setAdminName] = useState('');
+  const [adminEmail, setAdminEmail] = useState('');
+  const volHandleChange = event => setVolInput(event.target.value);
+  const adminNameHandleChange = event => setAdminName(event.target.value);
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'adminNameHandleChange' is assigned a value but never used.